<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ny_copySDK_window</class>
 <widget class="QMainWindow" name="ny_copySDK_window">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>233</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>424</width>
    <height>233</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>424</width>
    <height>233</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ny_copySDK</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>424</width>
     <height>108</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>424</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>10</y>
      <width>132</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="attrText_LYT">
     <item>
      <widget class="QLabel" name="source_TXT">
       <property name="text">
        <string>Source Driver Attribute:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="destination_TXT">
       <property name="text">
        <string>New Driver Attribute:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>10</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="textBox_LYT">
     <item>
      <widget class="QLineEdit" name="source_TB">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="destination_TB">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="setBtn_LYT">
     <item>
      <widget class="QPushButton" name="source_BTN">
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="destination_BTN">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>351</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="copyButton_LYT">
     <item>
      <widget class="QPushButton" name="copy_BTN">
       <property name="text">
        <string>COPY!</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="drvnObj_LBL">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Driven Object</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="drvnObj_LYT">
     <item>
      <widget class="QLabel" name="obj_s4_LBL">
       <property name="text">
        <string>Search For:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="obj_s4_TB"/>
     </item>
     <item>
      <widget class="QLabel" name="obj_rw_LBL">
       <property name="text">
        <string>Replace With:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="obj_rw_TB"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="drvnAttr_LYT">
     <item>
      <widget class="QLabel" name="attr_s4_LBL">
       <property name="text">
        <string>Search For:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="attr_s4_TB"/>
     </item>
     <item>
      <widget class="QLabel" name="attr_rw_LBL">
       <property name="text">
        <string>Replace With:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="attr_rw_TB"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="drvnAtr_LBL">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Driven Attribute (Optional)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>424</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
